app/Http/Controllers/Auth/LoginController.php

```diff
@@ -24,7 +27,18 @@ public function __invoke(Request $request): Response
             return [$provider => config("services.$provider.name") ?? __("auth.login_provider_{$provider}")];
         });
 
-        return Inertia::render('Auth/Login', [
+        $webauthnRemember = $request->cookie('webauthn_remember');
+        $data = [];
+        if ($webauthnRemember) {
+            if (($user = User::find($webauthnRemember)) && $user->webauthnKeys()->count() > 0) {
```

Comment:
Are you sure that works?
You instantiate the $user variable and right after, you query it with `$user->something`? If the user can't be found, the latter won't work.

Reply:
Yes it is working, the first parenthesis `($user = ...)` is evaluated before, and if the user is not found, then the `if` result is `false`.